projects/snapshot_bot/snapshot_bot.cc

```diff
@@ -513,13 +416,13 @@ class RobotFSM : FSM<State>::StateHandler
                 // If A is pressed
                 if(m_Joystick.isPressed(HID::JButton::A)) {
                     // Subtract time we've already moved for from drive time
-                    m_DriveTime -= m_MoveStopwatch.elapsed();
+                    m_DriveTime -= static_cast<millisecond_t>(m_MoveStopwatch.elapsed());
 
                     // Transition to correct paused state
                     m_StateMachine.transition((state == State::DrivingForward) ? State::PausedDrivingForward : State::PausedTurning);
                 }
                 // Otherwise, if drive time has passed
-                else if(m_MoveStopwatch.elapsed() > m_DriveTime) {
+                else if(static_cast<millisecond_t>(m_MoveStopwatch.elapsed()) > m_DriveTime) {
```

Comment:
Likewise

Reply:
If it doesn't and it's using ``std::chrono`` types, why not just stick with those - they're not used in complex units-based calculations or anything!

Reply:
Good point. I sort of hoped that it would end up making the code tidier but that didn't pan out...